content/source/docs/cloud/workspaces/managing-variables.html.md

```diff
@@ -0,0 +1,186 @@
+
+6. Click **Create variable set.** Terraform Cloud adds the new variable set to any specified workspaces and displays it on the **Variable Sets** page.
+
+### Edit Variable Sets
+
+To edit or remove variable sets:
+
+1. Click **Settings** in the top menu bar and then click **Variable Sets** in the left sidebar. The **Create a new Variable set** page appears.
+2. Click the variable set you want to edit. The **Edit Variables** page appears, where you can change the variable set settings. Refer to [create variable sets](#create-variable-sets) for details.
+
+### Delete Variable Sets
+
+To delete the variable set:
+
+1. Click **Settings** in the top menu bar and then click **Variable Sets** in the left sidebar. The **Create a new Variable set** page appears.
+2. Select **Delete variable set**. Terraform Cloud removes the variable set from all workspaces.
+
+### Apply or Remove Variable Sets From Inside a Workspace
+
+To apply a variable set to a specific workspace:
+
+1. Go to the workspace and click the **Variables** tab. The **Variables** page appears, showing all workspace-specific variables and variable sets applied to the workspace.
+
+2. Click **Apply Variable Set**, select the variable set you want to apply to your workspace, and click **Apply variable set**. The new variable set appears in the list and the Terraform will use the variables in the set during the next workspace run.
+
+To remove a variable set from within a workspace:
+
+1. Go to the workspace and click the **Variables** tab. The **Variables** page appears, showing all workspace-specific variables and variable sets applied to the workspace.
+2. Click the ellipses button next to the variable set and select **Remove variable set**.
+3. Click **Remove variable set** in the dialog box. Terraform Cloud removes the variable set from this workspace, but it remains available to other workspaces in the organization.
+
+## Overwrite Variable Sets
+
+You can overwrite variables defined in variable sets within a workspace. For example, you may want to use a different set of provider credentials in a specific workspace.
+
+To overwrite a variable from a variable set, [create a new workspace-specific variable](#workspace-specific-variables) of the same type with the same key. Terraform Cloud marks any variables that you overwrite with a yellow exclamation point and provides a link to the variable that will take precedence during runs.
+
+![An overwritten variable marked with a yellow exclamation point](link)
+
+Variables within a variable set can also automatically overwrite, or are overwritten by, variables with the same key in other variable sets applied to the workspace. Refer to [variable precedence](/docs/cloud/workspaces/variables.html#precedence) for more details.
```

Comment:
```suggestion
Variables within a variable set can also automatically overwrite variables with the same key in other variable sets applied to the same workspace. Refer to [variable precedence](/docs/cloud/workspaces/variables.html#precedence) for more details.
```

Reply:
Also, it might be good to be explicit about the fact that even though variable sets themselves are an organization-level object, overwriting only happens at the level of the workspace.

Reply:
good call! I'll call that out.